scribejava-apis/src/main/java/com/github/scribejava/apis/GeniusApi.java

```diff
@@ -0,0 +1,60 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.apis.service.GeniusOAuthServiceImpl;
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.oauth.OAuth20Service;
+import com.github.scribejava.core.utils.OAuthEncoder;
+
+public class GeniusApi extends DefaultApi20 {
+	
+	private static final String AUTHORIZE_URL =
+			"https://api.genius.com/oauth/authorize?"
+			+ "client_id=%s&"
+			+ "redirect_uri=%s&"
+			+ "scope=%s&"
+			+ "state=%s&"
+			+ "response_type=code";
+	
+	private static final String ACCESS_TOKEN_URL =
+			"https://YOUR_REDIRECT_URI/?"
+			+ "code=CODE&"
+			+ "state=SOME_STATE_VALUE";
```

Comment:
It's oauth2 base
http://tools.ietf.org/html/rfc6749#section-4.1.2
It can be placed somewhere in javadocs. But do not think that's part of specification will be useful be copied in javadoc, actually.